### PR TITLE
[15.0][MIG] crm_industry

### PR DESCRIPTION
**Pull request by SilvioC2C**
Depends on https://github.com/OCA/partner-contact/pull/1193

### CODE REVIEW

**Review by leemannd — APPROVED** [review]
Technicall LGTM, but runbot is failing

**Review by i-vyshnevska — APPROVED** [review]

**Comment by SilvioC2C** [comment]
> Technicall LGTM, but runbot is failing

@leemannd I don't understand why tho :disappointed: any help?

**Comment by leemannd** [comment]
@SilvioC2C I quickly checked on other repos. From what I see runbot is still not working on 15.0 branches. So nothing to be done on your side

**Comment by i-vyshnevska** [comment]
@SilvioC2C  you were depending on not merged addon without having a branch as a dependency 
rebase start to build rebase and push

**Comment by i-vyshnevska** [comment]
note for future the way how dependency is added changed for v 15.0I hadn't see docs for that FTM 
example 
https://github.com/OCA/edi/blob/f587eaae4ef977ced0373495b41c4c9e6be6a288/test-requirements.txt

**Comment by SilvioC2C** [comment]
@i-vyshnevska I rebased and force-pushed here: https://github.com/OCA/crm/pull/413#event-5741290075
At that point, the other PR was already merged: https://github.com/OCA/partner-contact/pull/1193#issuecomment-989904351

I will just update the `test-requirements.txt` file and force-push again, let's see if it's enough.

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
+partner-contact https://github.com/OCA/partner-contact 15.0
```
Comment (ivantodorovich) [inline oca_dependencies.txt]: This is not used anymore AFAIK

Comment (SilvioC2C) [reply oca_dependencies.txt]: @ivantodorovich should I remove the whole file?

Comment (simahawk) [reply oca_dependencies.txt]: YEEEEEEEEEEEEEEEEEEEES :tada: :partying_face:

**Review by simahawk — APPROVED** [review]
POT files should be skipped but that's not a blocker.
Next time consider giving a try to https://github.com/OCA/maintainer-tools/pull/504 :wink:

**Pull request merged**